admin-client/src/components/Alert.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
```

Comment:
Moved this to it's own component so the `onMount` functionality actually fires correctly!